videostore/encoder.go

```diff
@@ -78,7 +79,7 @@ func (e *encoder) initialize() error {
 
 	if ret != C.VIDEO_STORE_ENCODER_RESP_OK {
 		err := errors.New("failed to initialize encoder")
-		e.logger.Errorf("%s: %d: %s", err.Error(), ret, ffmpegError(ret))
+		e.logger.Errorf("%s: %d: %s", err.Error(), ret, vsutils.FFmpegError(int(ret)))
```

Comment:
had to change the param of `ffmpegError` which is now `vsutils.FFmpegError` to take in a Go int due to `cannot use C.int(ret) (value of type _Ctype_int) as vsutils._Ctype_int value in argument to vsutils.FFmpegError`